src/app/widgets/reactive-fields/reactive-label/reactive-label.component.html

```diff
@@ -0,0 +1,11 @@
+<label class="label d-flex flex-column gap-1">
+  <div *ngIf="label" [class.font-weight-bold]="isBold">
+    {{ label }}
+    <span *ngIf="hasRequiredValidator" class="label__asterisk"> *</span>
+    <span *ngIf="hint && hintPosition === 'top'" class="label__hint" [ngStyle]="hintStyle">{{ hint }}</span>
```

Comment:
I think we need to stay with classes here, because of color variables

![image](https://user-images.githubusercontent.com/55241607/231358359-3faab76f-761a-422d-a8a3-cecb232dbcf7.png)